combat-trainer.lic

```diff
@@ -1055,15 +1055,15 @@ class TrainerProcess
     dead_count = DRRoom.dead_npcs.size
 
     EquipmentManager.instance.stow_weapon(game_state.weapon_name)
-    EquipmentManager.instance.wield_weapon(game_state.ambush_stun_weapon, 'Small Blunt')
+    EquipmentManager.instance.wield_weapon(@stun_weapon)
```

Comment:
You'll still need to pass in a weapon skill here. If the `@stun_weapon` is swappable, `equipmanager` needs to know which skill it should use.